app/src/main/java/com/framgia/wsm/data/source/RequestRepository.java

```diff
@@ -131,7 +131,7 @@ public RequestRepository(RequestRemoteDataSource remoteDataSource) {
         requestOverTime4.setCreatedAt("23/08/2017");
         requestOverTime4.setFromTime("21/08/2017 18:00");
         requestOverTime4.setToTime("22/08/2017 19:00");
-        requestOverTime4.setStatus(0);
+        requestOverTime4.setStatus("approved");
```

Comment:
ERROR: [WrongConstant] Must be one of: StatusCode.ACCEPT_CODE, StatusCode.PENDING_CODE, StatusCode.REJECT_CODE
 Ensures that when parameter in a method only allows a specific set of constants, calls obey those rules.